master/src/main/scala/Main.scala

```diff
@@ -1,7 +1,43 @@
 package kr.ac.postech.paranode.master
 
+import kr.ac.postech.paranode.core.WorkerMetadata
+import kr.ac.postech.paranode.master.AuxFunction.getWorkerDetails
+import kr.ac.postech.paranode.rpc.MasterServer
+
+import java.net.URL
+import scala.io.Source
+import scala.util.Try
+
 object Main {
   def main(args: Array[String]): Unit = {
-    println("Hello world!")
+    val requestLimit = Try(args(0).toInt).getOrElse {
+      println("Invalid command")
+      return
+    }
+
+    val server = new MasterServer(scala.concurrent.ExecutionContext.global)
+    server.startServer()
+
+    while (server.getRequestCount < requestLimit) {
+      Thread.sleep(1000)
+    }
+
+    val workerInfo: List[WorkerMetadata] = getWorkerDetails()
+
+    assert(workerInfo.size == requestLimit)
+
+    try {
+      val url = new URL("http://checkip.amazonaws.com")
+      val source = Source.fromURL(url)
+      val publicIpAddress = source.mkString.trim
+      source.close()
+
+      println(publicIpAddress + ":" + server.getPort)
```

Comment:
`InetAddress.getLocalHost.getHostAddress` makes sense. Getting machine ip from outside is not a good idea. e.g. if we are using internal network, it should make troubles.

Reply:
https://github.com/betarixm/434project/pull/71/commits/b5db92ab7c09755f9de355b1f7906e129cf55542
https://github.com/betarixm/434project/pull/71/commits/5e04944846cc46b9880281e3de16844b7da49177